builtin/logical/aws/backend_test.go

```diff
@@ -1286,6 +1286,36 @@ func TestBackend_FederationTokenWithGroups(t *testing.T) {
 	})
 }
 
+func TestBackend_SessionToken(t *testing.T) {
```

Comment:
I couldn't get _any_ tests to pass locally for whatever reason, so while I did test this PR, I'm not sure if this test specifically is valid and passes.